src/controllers/user.controller.js

```diff
@@ -1 +1,27 @@
-exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
\ No newline at end of file
+const User = require('../models/user.model')
+const errorHandler = require('../middlewares/error-handler')
+
+exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
+
+exports.add_user = async (req, res, next) => {
+  const { title, username, firstname, lastname, graduation } = req.body;
+  const newUser = {
+    profile_picture:'update.to.s3.default',
+    title: [title],
+    badges: ['General'],
+    firstname,
+    lastname,
+    username,
+    graduation,
+    description: '',
+  }
+
+  User.create(newUser, (err, user) => {
+    if (err) {
+      console.log(err)
```

Comment:
use the logging middleware instead of console.log, gives us concrete log files with proper log levelling

Reply:
Probably will create a new middleware for logging will add an action to write this.

Reply:
there is logging middleware making one is a ticket in and of itself.